code/modules/fish/fishtank.dm

```diff
@@ -30,6 +30,8 @@
 	var/leaking = FALSE			// 0 if not leaking, 1 if minor leak, 2 if major leak (not leaking by default)
 	var/shard_count = 0			// Number of glass shards to salvage when broken (1 less than the number of sheets to build the tank)
 
+	var/mob/current_fisher = null // Only one person can fish in the tank at a time
```

Comment:
```suggestion
	///The Mob that is currently fishing in the fish tank
	var/mob/current_fisher 
```
if you don't assign a variable at definition, it should be null regardless. Also document variables like this for the auto doc

As a side note, make sure this will Garbage Collect properly, if the current_fisher is gibbed, dusted, cryo'd etc, will this reference be properly removed?